pyne_ubuntu.16.04.py2.dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM ubuntu:18.04
+
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+RUN apt-get clean -y
+
+RUN apt-get update
+RUN apt-get update
+RUN apt-get install -y --fix-missing \
+    software-properties-common wget g++ \
+    build-essential python-numpy python-scipy cython python-setuptools \
+    python-nose git cmake vim emacs gfortran libblas-dev \
+    liblapack-dev libhdf5-dev libhdf5-serial-dev gfortran python-tables \
+    python-matplotlib python-jinja2 python-dev libpython-dev \
+    autoconf libtool python-setuptools python-pip doxygen 
+RUN apt-get clean -y
+                       
+RUN pip install sphinx \
+    cloud_sptheme \
+    prettytable \
+    sphinxcontrib_bibtex \
+    numpydoc \
+    nbconvert \
+    numpy 
+
+# make starting directory
+RUN mkdir -p $HOME/opt
+RUN echo "export PATH=$HOME/.local/bin:\$PATH" >> ~/.bashrc \
+    && echo "alias build_pyne='python setup.py install --user -j 3 -DMOAB_LIBRARY=\$HOME/opt/moab/lib -DMOAB_INCLUDE_DIR=\$HOME/opt/moab/include'" >> ~/.bashrc \
+    && echo "alias python=python" >> ~/.bashrc \
+    && echo "alias nosetests=nosetests3" >> ~/.bashrc
+
+RUN gcc --version
+
+# build MOAB
+RUN cd $HOME/opt \
+  && mkdir moab \
+  && cd moab \
+  && git clone https://bitbucket.org/fathomteam/moab \
+  && cd moab \
+  && git checkout -b Version5.0 origin/Version5.0 \
```

Comment:
I thought we used version 5.1.0

Reply:
Yes my bad, I tried `Version5.1`, but it does not exist as the named it `Version5.1.0`...
updating that...